iconrpcserver/utils/json_rpc.py

```diff
@@ -52,14 +52,15 @@ def process_response(self, response, log_extra=None, log_format=None):
             # Log the response before processing it
             self.log_response(response, log_extra, log_format)
             # If it's a json string, parse to object
-            if isinstance(response, basestring):
+            if isinstance(response, str) or isinstance(response, bytes):
```

Comment:
How about using `AnyStr`?
https://docs.python.org/3/library/typing.html#typing.AnyStr

Reply:
Can you explain usage of `typing.AnyStr` in `isinstace` built-in method?

I tried:
```
Python 3.7.3 (default, Dec 10 2019, 01:42:03)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.10.2 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from typing import AnyStr

In [2]: a = "a"

In [3]: isinstance(a, AnyStr)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-a31d16311937> in <module>
----> 1 isinstance(a, AnyStr)

TypeError: isinstance() arg 2 must be a type or tuple of types

...

In [6]: b = b"b"

In [7]: isinstance(b, AnyStr)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-23dc579345ee> in <module>
----> 1 isinstance(b, AnyStr)

TypeError: isinstance() arg 2 must be a type or tuple of types
```

Reply:
Oops Sorry
In this case (bytes, str) is correct